src/components/CollaboratorModal/submodals/RemoveCollaboratorSubmodal.tsx

```diff
@@ -1,35 +1,50 @@
-import { ModalHeader, ModalBody, Text, Stack } from "@chakra-ui/react"
+import {
+  ModalHeader,
+  ModalBody,
+  Text,
+  Stack,
+  useModalContext,
+} from "@chakra-ui/react"
 import { Button, ModalCloseButton } from "@opengovsg/design-system-react"
 import { useCollaboratorModalContext } from "components/CollaboratorModal/CollaboratorModalContext"
-import { CollaboratorModalState } from "components/CollaboratorModal/constants"
 
+import { useLoginContext } from "contexts/LoginContext"
+
+import { useDeleteCollaboratorHook } from "hooks/collaboratorHooks"
 import useRedirectHook from "hooks/useRedirectHook"
 
-const RemoveCollaboratorSubmodal = () => {
+const TEXT_FONT_SIZE = "14px"
```

Comment:
hmm can we just remove this constant? feels like its only being used at one place only, seems trivial enough to be just used directly in the code? 
Nit: let's shift this to rem instead!

Reply:
think i'lkl shift this to `constants` and change it to rem as it's being used in `AcknowledgementSubmodal` and `RemoveCollaboratorSubmodal`